docs/en/ingest-management/elastic-agent/elastic-agent-standalone-logging.asciidoc

```diff
@@ -0,0 +1,126 @@
+[[elastic-agent-standalone-logging-config]]
+[role="xpack"]
+= Configure logging for {agent} standalone
```

Comment:
```suggestion
= Configure logging for standalone {agent}s
```

Reply:
I'd say either "standalone Elastic Agents" or "Elastic Agents running standalone"

Reply:
Got it. Thanks!